lightning_pose/utils/plotting_utils.py

```diff
@@ -134,9 +98,45 @@ def predict_videos(
         cfg = cfg_file
     else:
         raise ValueError("cfg_file must be str or DictConfig, not %s!" % type(cfg_file))
+    return cfg
+
+
+def check_prediction_file_format(save_file: str) -> None:
+    if not (
+        save_file.endswith(".csv")
+        or save_file.endswith(".hdf5")
+        or save_file.endswith(".hdf")
+        or save_file.endswith(".h5")
+        or save_file.endswith(".h")
+    ):
+        raise NotImplementedError("Currently only .csv and .h5 files are supported")
+
+
+@typechecked
+def predict_single_video(
+    video_file: str,
+    ckpt_file: str,
+    cfg_file: Union[str, DictConfig],
+    preds_file: Optional[str] = None,
+    sequence_length: int = 16,
+    device: Literal["gpu", "cuda", "cpu"] = "gpu",
+    video_pipe_kwargs: dict = {},
+):
```

Comment:
i keep forgetting what the differences are between all these predict functions; could add a simple docstring like """Predict poses on a single video from a config and model checkpoint file."""

Reply:
I added detailed docstrings. If you have 10-15 minutes, it'll be good if you can to take a look at these funcs yourself too. there are some redundant arguments there `data_name`, `save_preds`. If you can help me polish it, it'll help.